grpc-proto/src/main/java/io/stargate/grpc/Values.java

```diff
@@ -232,9 +236,7 @@ public static double double_(Value value) {
   public static ByteBuffer byteBuffer(Value value) {
     checkInnerCase(value, InnerCase.BYTES);
 
-    ByteBuffer bytes = ByteBuffer.allocate(value.getBytes().size());
-    value.getBytes().copyTo(bytes);
-    return bytes;
+    return value.getBytes().asReadOnlyByteBuffer();
```

Comment:
Makes sense, as per issue description. Would benefit from Javadoc stating it too since call sequence does not immediately suggest (at least to me) the behavior.